app.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Lair",
+  "description": "Lair is a reactive attack collaboration framework and web application built with meteor",
+  "repository": "git://github.com/ilyaglow/lair.git#heroku-button",
```

Comment:
Yes, surely. I was just too lazy to add/manage an additional branch for the PR, so I can test it and stuff.